civihr_employee_portal/civihr_leave_absences/templates/my_leave.html

```diff
@@ -11,29 +11,574 @@
+                              </div>
+                          </div>
+                      </div>
+
+                      <div class="chr-separator"></div>
+
+                      <div class="row  chr-container__row">
+                          <div class="col-xs-12">
+                              <div class="table-responsive">
+                                  <table class="table table-borderless">
+                                      <thead>
+                                        <tr>
+                                            <th colspan="2"></th>
+                                            <th colspan="2">Dates</th>
+                                            <th colspan="2">Status</th>
+                                            <th>Vacation</th>
+                                            <th>TOIL</th>
+                                            <th>Maternity</th>
+                                            <th>Sick</th>
+                                            <th>Other</th>
+                                            <th></th>
+                                        </tr>
+                                      </thead>
+                                  </table>
+                              </div>
+                          </div>
+                      </div>
+
+                      <div class="chr-separator"></div>
+
+                        <div class="row panel panel-default chr-grid">
+                            <div class="panel-heading chr-grid__panel-heading">
+                                <h4 class="panel-title">
+                                    <div class="row ">
+                                        <div class="table-responsive">
+                                            <table class="table table-borderless">
+                                                <thead>
+                                                  <tr class="civi-grid--open ">
+                                                      <th colspan="2"><i class="fa fa-chevron-down" aria-hidden="true"></i> Period Entitlement</th>
+                                                      <th colspan="2"></th>
+                                                      <th></th>
+                                                      <th>31</th>
+                                                      <th>3</th>
+                                                      <th>0</th>
+                                                      <th>0</th>
+                                                      <th>0</th>
+                                                  </tr>
+                                                </thead>
+                                            </table>
+                                        </div>
+                                    </div>
+
+                                </h4>
+                            </div>
+                            <div id="collapseOne" class="panel-collapse collapse in">
```

Comment:
again, you are using camelCase for the `[id]`, but you should use the dasherized-case